Integrations/integration-MISP_V2.yml

```diff
@@ -456,6 +456,11 @@ script:
 
         event = MISP.new_event(**event_dic)
         event_id = event.get('id')
+        try:
```

Comment:
Don't really like this way of handling, why not just `if event_id.isdigit():` ?

Reply:
because it's more pythonic `The Python community uses an EAFP (easier to ask for forgiveness than permission) coding style. This coding style assumes that needed variables, files, etc. exist. Any problems are caught as exceptions. This results in a generally clean and concise style containing a lot of try and except statements` while using ifs is more jsic.
But I can still change it tho

Reply:
yes so please change